content/reference/user/artifact_gen/index.md

```diff
@@ -1,33 +1,35 @@
 ---
 title: "Artifact Generation"
-description: "Learn how to generate an artifact for FrontLine from Gatling zip bundle or a maven, SBT or gradle project."
-lead: "Generate an artifact from your Gatling bundle or with a maven, SBT or gradle project."
+description: "Learn how to generate an artifact for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT or Gradle project."
+lead: "Generate an artifact from your Gatling bundle or with a Maven, SBT or Gradle project."
 date: 2021-03-08T13:50:32+01:00
 lastmod: 2021-03-08T13:50:32+01:00
 weight: 10040
 ---
 
 ## Generating Artifacts for FrontLine
 
-FrontLine deploys packages containing your compiled Simulations and resources.
+Gatling Enterprise deploys packages containing your compiled Simulations and resources.
 Those packages have to be generated upstream.
-FrontLine is compatible with Gatling 3.3, 3.4 and 3.5.
+Gatling Enterprise is compatible with Gatling 3.3, 3.4, 3.5 and 3.6.
 
 ### Gatling zip bundle
 
-Please copy the [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) or [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) files in the `bin` directory of your Gatling unzipped bundle.
+Copy the [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) or [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) file in the `bin` directory of your unzipped Gatling bundle.
```

Comment:
Already present in gatling bundle 3.6.x
We can remove that

Reply:
Done; I added a warning that 3.6.0+ is required in this section.